packages/flutter_tools/lib/src/vscode/vscode_validator.dart

```diff
@@ -23,13 +23,20 @@ class VsCodeValidator extends DoctorValidator {
 
   @override
   Future<ValidationResult> validate() async {
-    final String? vsCodeVersionText = _vsCode.version == null
-        ? null
+    final List<ValidationMessage> validationMessages =
+      List<ValidationMessage>.from(_vsCode.validationMessages);
+
+    final String vsCodeVersionText = _vsCode.version == null
+        ? 'unknown version'
         : userMessages.vsCodeVersion(_vsCode.version.toString());
 
```

Comment:
Same thing here if the strings end up being the same thing

```suggestion
    final String vsCodeVersionText = _vsCode.version == null
        ? userMessages.vsCodeVersion('unknown')
        : userMessages.vsCodeVersion(_vsCode.version.toString());

```

Reply:
Also, small nit but noticed that we are accessing `userMessages` differently between Android Studio (injected in constructor) and VS Code (referencing top level variable in `user_messages.dart`).